pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -305,7 +305,7 @@ func (k *Bootstrapper) WaitCluster(k8s config.KubernetesConfig) error {
 	// by a CNI plugin which is usually started after minikube has been brought
 	// up. Otherwise, minikube won't start, as "k8s-app" pods are not ready.
 	componentsOnly := k8s.NetworkPlugin == "cni"
-	out.T(out.WaitingPods, "Verifying:")
+	out.T(out.WaitingPods, "Waiting for pods:")
```

Comment:
I am not sure if waiting for pods ! is newbie friendly !

Reply:
Do you have an alternative recommendation? 

I chose the word 'pods' for both accuracy and as a teaching tool for Kubernetes. I figured it was a good time to introduce it into their vocabulary if it wasn't there already.

Without the word 'pods', it's hard for them to figure out how to use 'kubectl' to dig further into the details.

Reply:
may be "Waiting for kuberernetes services to be up"

Reply:
Service is already used elsewhere in Kubernetes: https://kubernetes.io/docs/concepts/services-networking/service/

Reply:
This now just says "Waiting for:"